app/internal/window/window.go

```diff
@@ -14,13 +14,12 @@ import (
 	"gioui.org/unit"
 )
 
-type Options struct {
-	Width, Height       unit.Value
-	MinWidth, MinHeight unit.Value
-	MaxWidth, MaxHeight unit.Value
-	Title               string
+type Option struct {
+	Width, Height       *unit.Value
```

Comment:
Combining these fields seems better in a Point type, analogous to image|f32.Point. Point belongs in package unit, but it's ok to put it in the app/internal/window package for now.